wordpress-mobile/WordPress-iOS
# Move Notification Settings to Notifications tab

This PR implements an item from the IA project – moving Notification Settings from the Me tab to the Notifications tab. Android PR is here: https://github.com/wordpress-mobile/WordPress-Android/pull/10335

![notifications](https://user-images.githubusercontent.com/4780/62711047-4dcbc400-b9f0-11e9-8e66-058f117507e8.png)

**To test:**

* Build and run
* Ensure that Notification Settings is no longer present in the Me tab
* Ensure that the cog icon is present in the Notifications screen when: you are signed into a wpcom site or a Jetpack site, but not when you are just signed into a non-Jetpack self-hosted site
* Tap the icon and ensure Notification Settings are displayed and usable
* Check universal links still work – with the simulator open, in Terminal run: `xcrun simctl openurl booted https://wordpress.com/me/notifications`. This should launch Notification Settings.

Update release notes:
